test/unit/traversing.js

```diff
@@ -743,6 +743,37 @@ QUnit.test( "contents()", function( assert ) {
 	assert.equal( c[ 0 ].nodeValue, "hi", "Check node,textnode,comment contents is just the one from span" );
 } );
 
+QUnit.test( "contents() for <template />", function( assert ) {
```

Comment:
As for the last test, it uses `assert.ok( true, "Something is not supported in this browser" )` pattern that we used before `QUnit.skip` was available. Now I'd rather expect something like:
```js
QUnit[ "content" in document.createElement( "template" ) ? "test" : "skip" ](
	"contents() for <template /> remains inert",
	function( assert ) {
		Globals.register( "testScript" );
		Globals.register( "testImgOnload" );
		/* the rest of the test */
	}
)();
```

Let me know if something is not clear.